dataviz/nuxeo-aggregate-data-behavior.js

```diff
@@ -363,7 +363,13 @@ Nuxeo.AggregateDataBehavior = {
 
   // unwrap out metrics value
   _getMetricsValue(bucket) {
-    return bucket.metrics ? bucket.metrics.value || bucket.metrics.values : bucket.doc_count;
+    if (bucket.metrics) {
+      if (bucket.metrics.value !== undefined) {
```

Comment:
nit: 

`return bucket.metrics.value !== undefined ? bucket.metrics.value : bucket.metrics.values;`

Reply:
I agree with this suggestion, it makes the code shorter without sacrificing legibility.